drm/ResourceManager.h

```diff
@@ -63,9 +63,13 @@ class ResourceManager {
   auto GetOrderedConnectors() -> std::vector<DrmConnector *>;
   void UpdateFrontendDisplays();
   void DetachAllFrontendDisplays();
+  void ReloadNode();
 
   std::vector<std::unique_ptr<DrmDevice>> drms_;
 
+  int card_num;
+  int callCount = 0;
```

Comment:
callCount is only used in function ResourceManager::UpdateFrontendDisplay, it seems we only need declare it as static varible, don't need declare it as class member. Also callCount should be named as call_count